net.tenshu.Terminator2.yml

```diff
@@ -0,0 +1,47 @@
+---
+id: net.tenshu.Terminator2
```

Comment:
`tenshu.net` doesn't show anything. is this upstream's?

Reply:
Correct, it's their dbus name used across the application

Reply:
I was talking about the domain name.

Reply:
so for some reason application refers to this old `net.tenshu.Terminator2` which no longer exists. The fork of original project is maintained at https://github.com/gnome-terminator/terminator so technically it should become the `com.github.gnome-terminator.terminator` right ? If I make this change, would this have bad effect on dbus session names as the `net.tenshu.Terminator2` is hardcoded within application itself.

Reply:
Also ... `error: 'com.github.gnome-terminator.terminator' is not a valid application name: Only last name segment can contain -`

Reply:
as per the doc, replace `-` with `_`. Also, as per the doc, it's `io.github` not `com.github`

Reply:
or even given that it links to `https://gnome-terminator.org/`, `org.gnome_terminator.Terminator2`

Reply:
ok so thats now renamed to org.gnome_terminator.Terminator2 and seems to be still working fine. Do I need to create new PR for this name ?